index.html

```diff
@@ -40,6 +40,10 @@ <h1>BTC</h1>
                 <div class="column">
                     :::
                 </div>
+                <div class="header">
+                    <h1>Practicing code review</h1>
+                </div>
+                <!-- <div class="comment"></div> -->
```

Comment:
Great job so far 💪  but it's not good practice to leave  commented code please remove this commented code ,this will make your code clean